core/server/services/channels/Channel.js

```diff
@@ -50,10 +49,22 @@ class Channel {
     }
 
     translateRoute(route) {
+        // TODO: find a better solution for this. Maybe integrated in urlService?
+        const routeKeywords = {
```

Comment:
Long Term:
- we don't need this anymore, because the route patterns are defined in the yaml file e.g. `/author/:slug/`
- you can define what you want in your yaml file e.g. `/person/:slug/`
- we no longer use e.g. `/:t_tag/:slug/`

Short Term (this PR)
- keep only the used keywords
  - only author and tag as far as i can see (see channels.json)

Reply:
> keep only the used keywords

Just thought about this again. We offered a custom channels file for beta testers.
But i would ignore this, because the old `config.channels.json` won't work at all with the new concept of yaml files.